### ISSUE TITLE
m-phase mutual inductor

### ISSUE THREAD

**Issue by modelica-trac-importer** [modelica/Modelica#173]
**Reported by AHaumer on 25 May 2009 14:26 UTC**
Suggestion for a m-phase mutual inductor:
v[m] = L[m,m]\*i[m]; where L[m,m]=
```
L1  M12 M13 ... M1m
M21 L2  M23 ... M2m
... ... ... ... ...
Mm1 Mm2 Mm3 ... Lm 
```
and for symmetry Mik=Mki.

----

Migrated-From: https://trac.modelica.org/Modelica/ticket/173

**Comment by modelica-trac-importer** [comment]
**Modified by AHaumer on 26 Mar 2010 07:46 UTC**

**Comment by modelica-trac-importer** [comment]
**Comment by christiankral on 7 May 2010 10:08 UTC**
We should include it right now because it is required for the Machines and FundamentalWave library

**Comment by modelica-trac-importer** [comment]
**Comment by AHaumer on 11 May 2010 14:14 UTC**
resolved by 98c87d388a29bd85c2213058495c6d79d226a4ad
